src/samples/WorkflowCore.Sample03/PassingDataWorkflow2.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using WorkflowCore.Interface;
+using WorkflowCore.Models;
+using WorkflowCore.Sample03.Steps;
+
+namespace WorkflowCore.Sample03
+{
+    public class PassingDataWorkflow2 : IWorkflow<Dictionary<string, int>>
+    {
+        public void Build(IWorkflowBuilder<Dictionary<string, int>> builder)
+        {
+            builder
+                .StartWith(context =>
+                {
+                    Console.WriteLine("Starting workflow...");
+                    return ExecutionResult.Next();
+                })
+                .Then<AddNumbers>()
+                .Input(step => step.Input1, data => data["Value1"])
+                .Input(step => step.Input2, data => data["Value2"])
+                .Output(data => data["Value3"], step => step.Output)
+                .Then<CustomMessage>()
```

Comment:
Indentation looks confusing

Reply:
Sure I will copy the indentation from the original file.